.travis.yml

```diff
@@ -1,6 +1,14 @@
 language: cpp
 sudo: required
 
+os:
+  - linux
+  - osx
+
+compiler:
+  - gcc
+  - clang
```

Comment:
I don't think we should set this here. Use the configure script to install the compiler you want and the build script to set CC and CXX calling the waf build system